src/index_runner/es_indexers/indexer_utils.py

```diff
@@ -77,35 +77,6 @@ def get_shared_users(ws_id):
     return shared_users
 
 
-def fetch_objects_in_workspace(ws_id, include_narrative=False):
```

Comment:
This is only used by the narrative indexer and is very specific to that, so I moved it inside that module